### ISSUE TITLE
[BUG] Option labels in black in Dark Theme

### ISSUE THREAD

**Issue by SalahAdDin** [node-vision/strapi-plugin-entity-relationship-chart#18]
Hello,

Thank you very much for this plugin, it is pretty cool for showing it to clients.

But we have a problem with it:
<img width="656" alt="Screen Shot 2022-08-08 at 02 20 10" src="https://user-images.githubusercontent.com/5034215/183314961-5a32c88c-7dd9-457f-b076-229b8dd9e248.png">

As you can see, in the dark theme, we can see the options, cause they are still in black.

How can we solve it?

Thank you!

**Comment by ShahriarKh** [comment]
https://github.com/node-vision/strapi-plugin-entity-relationship-chart/blob/337686ee92b3d8a9b00ee393fe69a086414bb143/admin/src/pages/HomePage/index.js#L142-L153

**Comment by SalahAdDin** [comment]
> The label's should have a color based on the current theme:
> 
> https://github.com/node-vision/strapi-plugin-entity-relationship-chart/blob/337686ee92b3d8a9b00ee393fe69a086414bb143/admin/src/pages/HomePage/index.js#L142-L153

It should, but it is not working, it seems this feature was not fully implemented.

**Comment by ShahriarKh** [comment]
Yeah I know. I meant that the label needs a dynamic color value; something like `<label style="color: var(--color-based-on-current-theme)" >`. However, I can't seem to find a way to get the current theme, and Strapi itself doesn't use css variables for this.

**Comment by ShahriarKh** [comment]
I checked other plugins sources, and they were using Strapi design system (https://design-system.strapi.io/) for components, so the colors and design matched the admin area perfectly.

**Comment by SalahAdDin** [comment]
It seems this plugin requires full refactoring.

**Comment by ShahriarKh** [comment]
https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-components-checkbox--base

**Comment by ShahriarKh** [comment]
> I checked other plugins sources, and they were using Strapi design system (https://design-system.strapi.io/) for components, so the colors and design matched the admin area perfectly.

https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-components-checkbox--base

**Comment by SalahAdDin** [comment]
> > I checked other plugins sources, and they were using Strapi design system (https://design-system.strapi.io/) for components, so the colors and design matched the admin area perfectly.
> 
> https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-components-checkbox--base
> 
> ```js
> import { Checkbox } from "@strapi/design-system";
> ```
> 
> ```js
> <Checkbox
>     checked={relations}
>     onValueChange={(e) => setRelations(e.target.checked)}
> >
>     relations
>     <Legend color="#ff6400" /> 
> </Checkbox>
> ```

Could you make a a pull request?
